<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>TextStyleDialog</class>
 <widget class="QDialog" name="TextStyleDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>584</width>
    <height>419</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="font">
   <font>
    <pointsize>10</pointsize>
   </font>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <property name="sizeGripEnabled">
   <bool>false</bool>
  </property>
  <widget class="QListWidget" name="item_widget">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>11</y>
     <width>171</width>
     <height>81</height>
    </rect>
   </property>
   <property name="editTriggers">
    <set>QAbstractItemView::NoEditTriggers</set>
   </property>
   <property name="textElideMode">
    <enum>Qt::ElideLeft</enum>
   </property>
   <item>
    <property name="text">
     <string>普通</string>
    </property>
    <property name="flags">
     <set>ItemIsSelectable|ItemIsUserCheckable|ItemIsEnabled</set>
    </property>
   </item>
   <item>
    <property name="text">
     <string>函数</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>关键字</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>数字</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>符号</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>预处理指令</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>注释</string>
    </property>
   </item>
  </widget>
  <widget class="QGroupBox" name="font_type">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>230</y>
     <width>171</width>
     <height>81</height>
    </rect>
   </property>
   <property name="title">
    <string>样式</string>
   </property>
   <widget class="QCheckBox" name="bold">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>20</y>
      <width>51</width>
      <height>25</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>10</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>黑体</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="italic">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>20</y>
      <width>51</width>
      <height>25</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>10</pointsize>
      <italic>true</italic>
     </font>
    </property>
    <property name="text">
     <string>斜体</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="underline">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>50</y>
      <width>61</width>
      <height>25</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>10</pointsize>
      <underline>true</underline>
     </font>
    </property>
    <property name="text">
     <string>下划线</string>
    </property>
   </widget>
  </widget>
  <widget class="QWidget" name="verticalLayoutWidget">
   <property name="geometry">
    <rect>
     <x>190</x>
     <y>10</y>
     <width>381</width>
     <height>311</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="text_layout">
    <property name="sizeConstraint">
     <enum>QLayout::SetDefaultConstraint</enum>
    </property>
   </layout>
  </widget>
  <widget class="QPushButton" name="OK_button">
   <property name="geometry">
    <rect>
     <x>330</x>
     <y>390</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>确定</string>
   </property>
  </widget>
  <widget class="QPushButton" name="cancel_button">
   <property name="geometry">
    <rect>
     <x>410</x>
     <y>390</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>取消</string>
   </property>
  </widget>
  <widget class="QPushButton" name="apply_button">
   <property name="geometry">
    <rect>
     <x>490</x>
     <y>390</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>应用</string>
   </property>
  </widget>
  <widget class="QGroupBox" name="theme_group">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>320</y>
     <width>171</width>
     <height>91</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="title">
    <string>主题</string>
   </property>
   <widget class="QPushButton" name="add_theme">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>20</y>
      <width>41</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text">
     <string>添加</string>
    </property>
   </widget>
   <widget class="QPushButton" name="remove_theme">
    <property name="geometry">
     <rect>
      <x>80</x>
      <y>20</y>
      <width>41</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text">
     <string>删除</string>
    </property>
   </widget>
   <widget class="QComboBox" name="theme_box">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>60</y>
      <width>101</width>
      <height>25</height>
     </rect>
    </property>
   </widget>
  </widget>
  <widget class="QGroupBox" name="color_group">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>100</y>
     <width>171</width>
     <height>121</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="title">
    <string>颜色</string>
   </property>
   <widget class="QComboBox" name="foreground_box">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>20</y>
      <width>101</width>
      <height>25</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="background_label">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>50</y>
      <width>31</width>
      <height>26</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>11</pointsize>
     </font>
    </property>
    <property name="text">
     <string>背景</string>
    </property>
   </widget>
   <widget class="QLabel" name="foreground_label">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>20</y>
      <width>31</width>
      <height>26</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>11</pointsize>
     </font>
    </property>
    <property name="text">
     <string>前景</string>
    </property>
   </widget>
   <widget class="QComboBox" name="background_box">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>50</y>
      <width>101</width>
      <height>25</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="add_color">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>85</y>
      <width>61</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text">
     <string>添加颜色</string>
    </property>
   </widget>
   <widget class="QPushButton" name="remove_color">
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>85</y>
      <width>61</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text">
     <string>删除颜色</string>
    </property>
   </widget>
  </widget>
  <widget class="QComboBox" name="font_family_box">
   <property name="geometry">
    <rect>
     <x>420</x>
     <y>340</y>
     <width>151</width>
     <height>25</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>10</pointsize>
    </font>
   </property>
  </widget>
  <widget class="QLabel" name="font_family_label">
   <property name="geometry">
    <rect>
     <x>380</x>
     <y>340</y>
     <width>31</width>
     <height>25</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="text">
    <string>字体</string>
   </property>
  </widget>
  <widget class="QLabel" name="font_size_label">
   <property name="geometry">
    <rect>
     <x>200</x>
     <y>340</y>
     <width>51</width>
     <height>25</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="text">
    <string>文字大小</string>
   </property>
  </widget>
  <widget class="QSpinBox" name="font_size_box">
   <property name="geometry">
    <rect>
     <x>270</x>
     <y>340</y>
     <width>61</width>
     <height>25</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>ADMUI3Lg</family>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="minimum">
    <number>9</number>
   </property>
   <property name="maximum">
    <number>50</number>
   </property>
   <property name="singleStep">
    <number>1</number>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
